src/components/tabs/README.md

```diff
@@ -303,8 +303,7 @@ If you want to add extra tabs that do not have any content, you can put them in
 <!-- b-tabs-item-slot.vue -->
 ```
 
-**Note:** extra (contentless) tab buttons should be a `<b-nav-item>` or have the class `nav-item`
-with a root element of `<li>` and class `nav-item` for proper rendering and semantic markup.
+**Note:** extra (contentless) tab buttons should be a `<b-nav-item>` or have a root element of `<li>` and class `nav-item` for proper rendering and semantic markup.
```

Comment:
Line needs to be wrapped at 100 characters to make prettier happy

Reply:
Done

Reply:
I should have been a bit more clear... "as many words that will fit into a 100 character width (with no trailing spaces)"

:wink: